src/pages/Rockets.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchAllRockets, toggleReservation } from '../redux/rockets/rocket';
+import './style.css';
+
+function Rockets() {
+  const rockets = useSelector((state) => state.rockets);
+  const dispatch = useDispatch();
+
+  const reserveRocket = (id) => {
+    dispatch(toggleReservation(id));
+  };
+
+  useEffect(() => {
+    dispatch(fetchAllRockets());
```

Comment:
- [ ] I think it's better to dispatch the action just once. Will create a more professional code and fix the issue of resetting all your reservations when reloading the rockets list.